### DISCUSSION TITLE
Confusion with the Request.body type

### DISCUSSION THREAD

**Discussion by berislavlopac** [p1c2u/openapi-core#824] [Q&A] [answered]
On this line: https://github.com/python-openapi/openapi-core/blob/44b8b3cb11158449be7646171cf0d8782415d02c/openapi_core/protocols.py#L20

the type of the `BaseRequest.body` property is defined as `Optional[bytes]`; however, the docstring of the `Request` protocol (which inherits `body` from `BaseRequest`) says:

>         body
>            The request body, as string.

I'm currently having some issue with the validation, but it's not clear which direction should I go with the solution -- should `body` be `bytes` or `str`? Have I missed something? Thank you!

**Comment by p1c2u** [comment, marked as answer]
Hi @berislavlopac 

that's definitely an issue with doctoring,`Optional[bytes]` is expected there

**Comment by berislavlopac** [comment]
All clear, thank you!
